ns3_adapter/include/ns3_client.h

```diff
@@ -24,7 +24,9 @@
 #include <thread>
 #include <queue>
 #include <vector>
-
+#include <rapidjson/document.h>
+#include <rapidjson/rapidjson.h>
+#include <optional>
```

Comment:
Are we using std::optional anywhere? If not this is unnecessary.

Reply:
`    std::optional<unsigned long> result;
    if (obj.HasMember(timestep_member_name.c_str()) && obj.FindMember(timestep_member_name.c_str())->value.IsUint64())
    {
        result = obj[timestep_member_name.c_str()].GetUint64();
    }
    ROS_DEBUG_STREAM("process_time deserialized unsigned long: " << std::to_string(result.value()));`
    
 It is used when extracting data member from json. I added more log to make it more meaningful. This is low chance to fail, but thought wouldn't hurt to add.

Reply:
We do not need to use optional. It is an unrelated functionality to rapidjson and used for things that can are optional (can be defined or not define). Timestep is not optional so using it here is a mistake.